communication/nats/dialog/dialog.go

```diff
@@ -2,13 +2,19 @@ package dialog
 
 import (
 	"github.com/mysterium/node/communication"
+	"github.com/mysterium/node/identity"
 )
 
 type dialog struct {
 	communication.Sender
 	communication.Receiver
+	peerID identity.Identity
 }
 
 func (dialog *dialog) Close() error {
 	return nil
 }
+
+func (dialog *dialog) GetPeerID() identity.Identity {
```

Comment:
Usual go convention is not to use get, just `PeerID()`

Reply:
Fixed